guide/matrix-vs-discord.md

```diff
@@ -74,32 +74,34 @@ Note that Matrix does not (and cannot, due to its decentralized nature) paywall
+| Publicity | Although Discord offers its own "server" discovery feature, the requirements are somewhat arbitrary, so third-party services are often used. Only "server" owners can apply to Discord's own "server" discovery directory. Third party directory services may differ. | Each homeserver has a room directory which anyone in that homeserver may publish to. |
 | Invite | Through generating invite links. | Through directly inviting users, or through shareable [addresses](../features/#promotion). |
-| Permissions | 255 roles. How long did it take for you to learn role hierarchy? | 2^54 power levels (I think it's -2^53 to 2^53-1, however I highly doubt you will *ever* reach that limit), with minimal permissions. A user acquires a permission if their power level is equal to or higher than the power level required for the specific permission. |
+| Permissions in group chats | Single-owner, up to 255 roles. How long did it take for you to learn role hierarchy?[^8] A "server" can deleted by its owner only, and that deletion affects everyone. | Up to 2^54 power levels (I think it's -2^53 to 2^53-1, however I highly doubt you will *ever* reach that limit), with minimal permissions. A user acquires a permission if their power level is equal to or higher than the power level required for the specific permission. Rooms are not owned by any user or server, hence cannot forcibly be shut down without coordination. |
+| Bans | Bans are only visible to "server" moderators. One can also see their own ban when they attempt to join a "server", but not the ban reason. | Bans are public to all members, along with the reasons. |
 | **Running a bot** | Running a bot in more than 100 "servers" requires proof of identity. Selfbotting is forbidden. | You can run bots on any user accounts[^5] [^6]. Selfbotting is permitted (but be nice). |
-| **Apps to access platform** | You are only allowed to use the official Discord app (including its PTB and Canary variants). Client modifications are forbidden but effectively tolerated. Third-party clients are forbidden. | Element is the main app, but [**you're welcomed to use whatever you wish**](../#what-app-should-i-use). You can even make an app yourself[^6]! |
+| **Apps to access platform** | You are only allowed to use the official Discord app (including its PTB and Canary variants). Client modifications are forbidden but effectively tolerated. Third-party clients are forbidden but some are also tolerated. | Element is the main app, but [**you're welcomed to use whatever you wish**](../#what-app-should-i-use). You can even make an app yourself[^6]! |
```

Comment:
> but some are also tolerated

Source?

Reply:
In https://news.ycombinator.com/item?id=25224151, Jake Henry says:

> We are not banning people for using Ripcord. We are banning people for using our API in ways that our official client doesn't. This is paramount to our spam detection strategy, which is detecting quirks in requests that try and emulate our official client, in order to send spam. There is a very thin line between "scripts meant to try and emulate our client to send spam in an automated fashion" and "third party client" both try and emulate our client, but perhaps for different intent. However, our systems currently treat these as equals. It is for this reason that we have maintained that 3rd party clients are against our ToS - and most warn you before you use them. Given the vanishingly small user-base of 3rd party clients relative to the tremendous amount of spam-bots that constantly are attacking our platform, it's an unfortunate reality that this is bound to happen.
> 
> If you are a real human, and not a spam bot, our general policy is to unban and tell you "hey don't do that again." Which is the outcome that should have (and eventually did) happened here.

Reply:
> There is a very thin line between [spam tools] and "third party client" both try and emulate our client, but perhaps for different intent. However, our systems currently treat these as equals.

So third-party clients that (try to) reimplement official client functions are practically outlawed? Because it's not possible to fully emulate the official client?

But client mod users usually don't get banned at all (provided that they are not engaging in other activities)

Reply:
> So third-party clients that (try to) reimplement official client functions are practically outlawed? 

The truth is their antispam has a high false positive due to the nature of the receiver curve (the more true positives, the more false positives), and they cannot reliably distinguish the two.

Reply:
So the reality is that third-party clients are banned _de facto_ while client mods that rely on the official client are not banned _de facto_ even if they're banned _de jure_, for what reason they ban it is irrelevant (especially in a practical guide)

Reply:
More like a third party client that implements _**certain**_ (but a constantly changing list of) features or manifests bot-like usage patterns are banned.

Reply:
```suggestion
| **Apps to access platform** | By ToS, you are only allowed to use the official Discord app (including its PTB and Canary variants). Some modifications are generally tolerated, however certain reimplementations may be forbidden. | Element is the main app, but [**you're welcomed to use whatever you wish**](../#what-app-should-i-use). You can even make an app yourself[^6]! |
```